PunchCard.py

```diff
@@ -56,14 +57,16 @@ def printWeekHours(hours, timeFormat):
         print('\nTotal hours for the week: {} hours {} minutes({} hours)'.format(intHours, intMinutes, floatHours))
 
 
-def main(input, timeFormat):  # pragma: no cover
+def main(config, timeFormat):  # pragma: no cover
+    daysOfTheWeek = ['saturday', 'sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday']
     weekHours = 0.0
-    print(input.pop(0))
-    for line in input:
-        daySplit = line.split(',')
-        dayLetter = daySplit.pop(0)[0]
-        dayHours = calculateDay(daySplit)
-        printDaysHours(dayLetter, dayHours, timeFormat)
+    print(config['title']+'\n')
+    for day in daysOfTheWeek:
+        if day not in config['day'] or config['day'][day] == {}:
```

Comment:
I think `not config['day'][day]` might be a bit more idiomatic—I could be wrong though